config/examples/Alfawise/U20/Configuration.h

```diff
@@ -75,12 +75,23 @@
 //
 // 1 - Select your Alfawise U30 or U20 or U20+ printer (NO MORE REQUIRED HERE)
 //     and the touchscreen version, either 1.1 or 1.2. Most recent in France are v1.2 (blue pcb)
-//     These defines are set in platformio.ini build parameters, sample for U20 -DU20 -DTS_V12
 //     U20_PLUS was not tested, as we do not have a printer to test. Print bed PID settings MUST be tuned for it.
 
-// Valid platformio.ini submodel values are U20_PLUS U20 U30 LK1 LK2 LK4
+// Valid submodels: U20, U20_PLUS, U30, LK1, LK1_PLUS, LK2, and LK4
 
-// Valid platformio.ini touchscreens are TS_V11 TS_V12 TS_V19
+#define U20
```

Comment:
Yea, its fine commented, and there is no real need to create 11 env... basically just 3 are required for the 3 bed sizes. But Longer3D also has a git and use their model names/strings... Ideally the default env should be the smallest bed (U30) to avoid problems if they dont flash the right one... They also made a new tiny one named CUBE2 with this board...